v2/internal/file/file.go

```diff
@@ -14,3 +17,18 @@ func Exists(filename string) (bool, error) {
 	}
 	return false, err
 }
+
+// EnsureDirectory checks if the dirPath directory exists. If not, it
+// creates it.
+func EnsureDirectory(dirPath string) error {
+	if dirExists, err := Exists(dirPath); err != nil {
+		return err
+	} else if !dirExists {
+		fmt.Println("Creating " + dirPath + " directory...")
```

Comment:
Sorry for not catching this one sooner.

As a general rule, we want to keep output like this _out_ of functions that have the potential for reuse. Supposing for instance that we ended up reusing this function somewhere server side-- this isn't something I want/need to be seeing show up in the server's logs.

What you might consider doing is returning a `bool` that indicates whether the directory was created (`true`) or not (`false`, since it already existed). Then the caller can act on that information and display such a message if it's contextually appropriate to do so.